lib/system.nim

```diff
@@ -3756,6 +3756,13 @@ proc instantiationInfo*(index = -1, fullPaths = false): tuple[
   ##     testException(IndexError, tester(1))
   ##     # --> Test failure at example.nim:20 with 'tester(1)'
 
+proc lineInfoToString*(file: string, line, column: int): string =
```

Comment:
There is no universe in which this procedure should be in the ``system`` module

Reply:
removed, see how I did it in updated PR; I managed to avoid code duplication (this is used in both system.nim and macros.nim) while at same time not exposing new procs in system.nim